services/traefik/26.0.0/defaults/cm.yaml

```diff
@@ -197,7 +197,7 @@ data:
     pilot:
       dashboard: false
     # This value should be equal to release name. Justification with timeline of changes:
-    # 1. This was set to `kommander-treafik` in an old chart version - https://github.com/traefik/traefik-helm-chart/blob/v10.30.1/traefik/templates/deployment.yaml#L42 (Shipped in DKP 2.4)
+    # 1. This was set to `kommander-treafik` in an old chart version - https://github.com/traefik/traefik-helm-chart/blob/v10.30.1/traefik/templates/deployment.yaml#L42 (Shipped in NKP 2.4)
```

Comment:
I was conflicted on changing things like this that were technically DKP to NKP. Let me know what you all think and if it makes sense to still have DKP here instead for anything prior to 3.0.

Reply:
maybe check with product?

Reply:
Confirmed with Sonali, it should remain DKP for past versions. Reverting this change.